src/App.jsx

```diff
@@ -107,17 +107,34 @@ function App() {
                             key={e.id}
                             todo={e}
                         >
-                            <Button
-                                name={''}
-                                className={'button__delete'}
-                                onClick={() => onClickCrossButton(e?.id)}
-                            />
+                            <SubMenu >
+
+                                <SubMenuButton
+                                    onClick={null}
+                                    name={'done'}
+                                >
+                                </SubMenuButton>
+
+                                <SubMenuButton
+                                    onClick={null}
+                                    name={'edit'}
+                                >
+                                </SubMenuButton>
+
+                                <SubMenuButton
+                                    onClick={() => onClickCrossButton(e?.id)}
+                                    name={'delete'}
+                                >
+                                </SubMenuButton>
+
+                            </SubMenu>
```

Comment:
First of all it should be moved to separate component
It also shouldn't be passed as children. Because implementing it this way you make your component not scalable:
  - in case if you want to pass other elements you have to rebuild your children every time
  - this new component should be part of TodoCard